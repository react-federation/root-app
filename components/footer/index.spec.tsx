import setup from 'utils/test/setup'
import Footer from '.'

describe('Footer', () => {
  it('should match snapshot', () => {
    const { asFragment } = setup(Footer)

    expect(asFragment()).toMatchSnapshot()
  })
})
