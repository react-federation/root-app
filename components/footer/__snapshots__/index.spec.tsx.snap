// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match snapshot 1`] = `
<DocumentFragment>
  .emotion-0 {
  padding-top: 4px;
  padding-bottom: 4px;
  background-color: gray.50;
  margin-top: 4px;
}

<div
    class="emotion-0"
  >
    <div
      class="container"
    >
      Made with ❤️ by The Prisma Architecture Team
    </div>
  </div>
</DocumentFragment>
`;
